js/blocks/BoxesBlocks.js

```diff
@@ -86,6 +86,32 @@ function setupBoxesBlocks() {
         }
     }
 
+    class DecrementOneBlock extends IncrementBlock {
+        constructor() {
+            super("decrementOne");
+            this.setPalette("boxes");
+            this.beginnerBlock(true);
+
+            this.setHelpString([
+                _("The Sub-1-from block subtracts one from the value stored in a box."),
```

Comment:
Could please add a translator note to say that "sub" is short for subtract?

Reply:
Or we can just spell out subtract here as well?

Reply:
Yes. Just spell it out.